src/components/Pagination.js

```diff
@@ -84,7 +84,7 @@ export default function Pagination(props) {
     return null;
   }
   return (
-    <div className="d-flex justify-content-center mt-2">
+    <>
```

Comment:
do you need this?

Reply:
thought I did...turned out again I don't :)

removed